string obstacle_name
string push_dir
float32 push_dist_m 

---
bool success
string message

---
string state
